test/performance/hamburg/build-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extraEdgesStopPlatformLink": true,
+  "stationTransfers": true,
+  "transitModelTimeZone": "Europe/Berlin",
+  "transitServiceStart": "2023-01-01",
+  "transitServiceEnd": "2023-12-01",
+  "platformEntriesLinking": true,
+  "subwayAccessTime": 0,
+  "netexDefaults": {
+    "groupFilePattern": "(.*)\\/.*\\.xml",
+    "feedId": "HH"
+  },
+  "osmDefaults": {
+    "osmTagMapping": "germany",
+    "timeZone": "Europe/Berlin"
+  },
+  "transitFeeds": [
+    {
+      "type": "netex",
+      "source": "https://geofoxdata.blob.core.windows.net/opentripplanner/NeTEx_Hamburg.zip",
+      "feedId": "HH"
+    }
+  ],
+  "osm": [
+    {
+      "source": "https://geofoxdata.blob.core.windows.net/opentripplanner/hvv_filtered.osm.pbf"
```

Comment:
If you don't mind I will copy these onto the same server that runs the speed test. That way we don't have to wait for the download for every run.

Reply:
Yes, feel free to do so! :)